samples/android/camera-calibration/src/org/opencv/samples/cameracalibration/CameraCalibrationActivity.java

```diff
@@ -96,13 +86,9 @@ public void onPause()
     public void onResume()
     {
         super.onResume();
-        if (!OpenCVLoader.initDebug()) {
-            Log.d(TAG, "Internal OpenCV library not found. Using OpenCV Manager for initialization");
-            OpenCVLoader.initAsync(OpenCVLoader.OPENCV_VERSION_3_0_0, this, mLoaderCallback);
-        } else {
-            Log.d(TAG, "OpenCV library found inside package. Using it!");
-            mLoaderCallback.onManagerConnected(LoaderCallbackInterface.SUCCESS);
-        }
+        mOpenCvCameraView.enableView();
```

Comment:
samples/android/15-puzzle/src/org/opencv/samples/puzzle15/Puzzle15Activity.java has check here.

We need to define a unified variant of code.